docs/src/docs/asciidoc/getting-started.adoc

```diff
@@ -146,12 +146,16 @@ the configuration are described in the following listings:
 .Gradle
 ----
 	plugins { <1>
-		id "org.asciidoctor.convert" version "1.5.9.2"
+		id "org.asciidoctor.jvm.convert" version "3.3.2"
+	}
+
+	configurations {
+		asciidoctorExt
```

Comment:
I think it'd be worth adding a callout that explains what's happening here.

Reply:
Done!